mjcf_to_sdformat/mjcf_to_sdformat/converters/link.py

```diff
@@ -94,7 +95,25 @@ def mjcf_geom_to_sdf(body, body_parent_name=None):
                                     inertial_euler.pitch(),
                                     inertial_euler.yaw()))
         link.set_inertial(inertial)
-
+    else:
+        try:
+            body_inertia = physics.named.model.body_inertia[body.name]
+            inertial = Inertiald(
+                MassMatrix3d(physics.named.model.body_mass[body.name],
+                             Vector3d(body_inertia[0],
+                                      body_inertia[1],
+                                      body_inertia[2]),
+                             Vector3d(0, 0, 0)),
+                Pose3d(physics.named.model.body_ipos[body.name][0],
+                       physics.named.model.body_ipos[body.name][1],
+                       physics.named.model.body_ipos[body.name][2],
+                       physics.named.model.body_iquat[body.name][0],
+                       physics.named.model.body_iquat[body.name][1],
+                       physics.named.model.body_iquat[body.name][2],
+                       physics.named.model.body_iquat[body.name][3]))
```

Comment:
nit: I would prefer to have a helper function in [sdformat_mjcf_utils/sdf_utils.py](https://github.com/gazebosim/gz-mujoco/blob/main/sdformat_mjcf_utils/sdformat_mjcf_utils/sdf_utils.py) that assists with this to ensure that we get the quaternion `wxyz` ordering right. Maybe `wxyz_list_to_quat`?

Reply:
https://github.com/gazebosim/gz-mujoco/pull/48/commits/a027f6ec1014990279f05a50272496a9fab06559